targets/TARGET_NXP/TARGET_LPC176X/TARGET_MBED_LPC1768/PinNames.h

```diff
@@ -106,11 +106,17 @@ typedef enum {
     A4 = P1_30,
     A5 = P1_31,
 
-    I2C_SCL = D15,
-    I2C_SDA = D14,
-
     // Not connected
-    NC = (int)0xFFFFFFFF
+    NC = (int)0xFFFFFFFF,
```

Comment:
Actually looked it up, the enum will just upgrade to uint64_t. It's up to you if you want it at the end.